TEST.md

```diff
@@ -2,34 +2,62 @@
 
 ## Quick start:
 
-_Note: If you haven’t, make sure to install Protractor globally with `npm install protractor@2.1.0 -g`._
-
-__Update Selenium Server binaries:__
-
-```sh
-webdriver-manager update
-```
-
-__Start Selenium Server:__
+Open a new Terminal window or tab and change to the project directory,
+then tell gulp to start the tests:
 
 ```sh
-webdriver-manager start
+gulp build
+gulp test:acceptance
 ```
 
-__Open a new tab and cd to the test directory, then tell protractor to start the tests:__
-
-```sh
-cd test/browser_tests
-protractor conf.js
-```
-
-If you want to test a server other than your local instance (assuming port 7000 is your local CFGov server) use the `--baseUrl` flag. Ex:
-
-```sh
-protractor conf.js --baseUrl=http://beta.consumerfinance.gov
-# or
-protractor conf.js --baseUrl=http://localhost:8000
-```
+If you want to test a server other than your local instance,
+edit the `HTTP_HOST` and `HTTP_PORT` values in your `.env` file
+and reload the settings with `cd .. && cd cfgov-refresh`. Type `y` if prompted.
+
+## Sauce Connect - send tests to the cloud
+
+Sauce Labs can be used to run tests remotely in the cloud.
+
+1. Log into [http://saucelabs.com/account](Log into http://saucelabs.com/account).
+2. [Download Sauce Connect](https://docs.saucelabs.com/reference/sauce-connect/#basic-setup)
+3. Open a new Terminal window or tab and navigate to the downloaded SauceConnect folder.
+   If you place the folder in your Application's folder this might look like:
+
+   ```
+   cd /Users/<YOUR MAC OSX USERNAME>/Applications/SauceConnect
+   ```
+4. Copy step 3 from the the SauceLabs
+   [Basic Setup instructions](https://docs.saucelabs.com/reference/sauce-connect/#basic-setup)
+   and run that in your Terminal window.
+   Once you see `Sauce Connect is up` in the Terminal,
+   that means the tunnel has successfully been established
+   > The Terminal command should already have your Sauce username and access key filled in.
```

Comment:
This quote isn't showing up correctly in the rendered file https://github.com/cfpb/cfgov-refresh/blob/flapjack/TEST.md#sauce-connect---send-tests-to-the-cloud